.github/workflows/initiate_release.yml

```diff
@@ -0,0 +1,47 @@
+name: Create release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The new version number with 'v' prefix. Example: v1.40.1"
+        required: true
```

Comment:
can we add a check to make sure the input is bigger than prev version?

Reply:
✅